tapdb/sqlc/migrations/000015_asset_witnesses.up.sql

```diff
@@ -0,0 +1,11 @@
+-- The witness index indicates the order of the witness in the list of witnesses
+-- for a given asset. We didn't really support more than one witness before, so
+-- the default value of 0 should be fine for all existing assets.
+ALTER TABLE asset_witnesses ADD COLUMN witness_index INTEGER NOT NULL DEFAULT 0;
```

Comment:
What about existing entries that don't have the `witness_index` field?

Reply:
That's explained in the comment :sweat_smile: